features/display-ruby.yml

```diff
@@ -0,0 +1,12 @@
+name: "display: ruby"
+description: "The `display: ruby` CSS declaration renders an element with the box layout of a `<ruby>` HTML element. Child elements may use equivalents to `<ruby>` internal elements such as `display: ruby-base` for `<rb>`."
+spec: https://drafts.csswg.org/css-display-3/#layout-specific-display
+group:
+  - css
+  - ruby
+compat_features:
+  - css.properties.display.ruby
+  - css.properties.display.ruby-base
+  - css.properties.display.ruby-base-container
+  - css.properties.display.ruby-text
+  - css.properties.display.ruby-text-container
```

Comment:
Documenting for posterity-

The HTML spec marks `<ruby-base>` and `<ruby-text-container>` as obsolete so I was looking into whether the CSS values should be deprecated (first in the CSS spec, then in BCD).

However, there is an open [issue](https://github.com/whatwg/html/issues/1771) to remove them from the list of obselete features. At some point this may see more implementation and need updates, but this is good as is.